pocs/camera/sbigudrv.py

```diff
@@ -213,35 +279,30 @@ def get_exposure_status(self, handle):
         query_status_results = QueryCommandStatusResults()
 
         with self._command_lock:
-            self._set_handle(handle)
+            self.set_handle(handle)
             self._send_command('CC_QUERY_COMMAND_STATUS',
                                params=query_status_params,
                                results=query_status_results)
 
         return statuses[query_status_results.status]
 
-    def take_exposure(self,
-                      handle,
-                      seconds,
-                      filename,
-                      exposure_event=None,
-                      dark=False,
-                      header=None):
-        """
-        Starts an exposure and spawns thread that will perform readout and write
-        to file when the exposure is complete.
-        """
-        ccd_info = self._ccd_info[handle]
-
+    def start_exposure(self,
```

Comment:
```suggestion
    def _start_exposure(self,
```

This should be the overridden private method, right?

Reply:
This is a `Driver` class not a `Camera` class so that private method isn't here. Instead this is the `Driver` method that the `_start_exposure` method of the corresponding `Camera` class class to actually tell the hardware to start the exposure. Makes more sense for this to be public as it's intended for external use (by `Camera`).